CONTRIBUTING.md

```diff
@@ -262,11 +545,176 @@ point, but don't worry. If you look at the branch you raised your
 Pull Request against (probably `master`), you should see a commit with
 your name on it. Congratulations and thanks for your contribution!
 
+### Reviewing Pull Requests
```

Comment:
CONTRIBUTING.md is for people who have never before (or very infrequently) contributed to Node.js. This section seems geared towards Collaborators and seems better suited for the COLLABORATORS_GUIDE.md doc. Are we just putting it here for visibility?

Reply:
It seems the message of this commit contains some explanation: https://github.com/nodejs/node/pull/15123/commits/daf0847cde8a029d16993a1a9ba4205cf7225820

Reply:
Yep, the commit log message contains the details. One bit of feedback that I have received from the community is that there is extremely little understanding about what to expect when a PR is opened. I've added this here to help ensure that the process is more visible.